spec/12-the-scala-standard-library.md

```diff
@@ -642,6 +642,12 @@ object Predef {
   def classOf[T]: Class[T] = null
    // this is a dummy, classOf is handled by compiler.
 
+  // valueOf -----------------------------------------------------------
+
+  /** Retrieve the single value of a type with a unique inhabitant. */
+  @inline def valueOf[T](implicit vt: ValueOf[T]): T {} = vt.value
```

Comment:
There are methods where the user is never supposed to be passing the evidence, like `classOf` or `new Array ...`, and `valueOf` seems to be a new addition. This new addition should be consistent with what we have already.

There will always be a conflict between making things bullet-proof and "it's just a standard method", but in the case of things living in `scala` they should absolutely be bullet-proof.